client/store/index.js

```diff
@@ -38,7 +38,6 @@ export default new Vuex.Store({
   getters: {
     ...mapGetters('repositories', ['repositories', 'repositoryQueryParams']),
     ...mapGetters('activities', ['activities']),
-    ...mapGetters('tes', ['tes']),
```

Comment:
@underscope @vladimyr 

I removed this because, from what I can tell, it is not used anywhere. I don't think any of them are used except the `repositories/repositories` one. I believe this was written so that we have a nested module structure but also have these getters globally available. If that is true then I would remove all of them including the repositories one.  The code that generates these getters is convoluted and I do not see a reason to keep it for just one. And if we do keep it I would recommend changing its name. It being named `mapGetters` is deceiving since that is the name of a `Vuex` method.

Reply:
👍 It's a relict of the past. Feel free to do so.